config/game_options.txt

```diff
@@ -393,6 +393,7 @@ ROUNDSTART_RACES human
 ## Races that are strictly worse than humans that could probably be turned on without balance concerns
 ROUNDSTART_RACES lizard
 #ROUNDSTART_RACES fly
+#ROUNDSTART_RACES moth
```

Comment:
Now just remove that `#` and we'll be merge-ready

Reply:
Shoo, shoo!

Reply:
![](https://i.imgur.com/TE9eN0R.jpg)

Reply:
>People still debating codebase config

I should remove the whole thing

Reply:
i should remove you